start-mycroft.sh

```diff
@@ -100,17 +100,17 @@ function launch-process() {
 function launch-background() {
     if ($first_time) ; then
         echo "Initializing..."
-        ${DIR}/scripts/prepare-msm.sh
+        "${DIR}/scripts/prepare-msm.sh"
         source-venv
         first_time=false
     fi
 
     name-to-script-path ${1}
 
-    # Check if already running
+    # Start (or restart if already running)
```

Comment:
I believe the old comment might be good to keep since `pgrep -f "python3 -m ${_module}" > /dev/null` isn't trivial to understand.

Reply:
Clarified